packages/types-dev/index.d.ts

```diff
@@ -323,6 +323,15 @@ declare global {
                 | 'fileToJsonl';
             [other: string]: string;
         }
+
+        /** Structure for notification actions */
+        export interface NotificationAction {
+            /** This message will be shown if instance is offline */
+            offlineMessage?: ioBroker.StringOrTranslated;
```

Comment:
Should we really add something like this here? This is relatively special for the case of "displaying something". Should this message not just be static and same for all and handled by the relevant UI??

Reply:
As more it standardized, as more easier to use it in e.g. admin.

Reply:
`additionalData` does not say anything about the object. But `actionData` explains that it is for action

Reply:
I would prefer `contextData` in the end this will be the object to provide any additional information for messages in the future.